graphs/util/db.py

```diff
@@ -4742,3 +4744,112 @@ def constructResponse(statusCode, message, error):
 		response['Error'] = error
 
 	return response
+
+
+def add_share_graph_event(graph_id, owner_id, group_id, member_id):
+	'''
+		Add a new share graph event to the table.
+		After sharing the graph with a group this function will create
+		a share graph event for all the users in that group
+
+		@param graph_id: id of the graph shared
+		@param owner_id: owner of the graph which is shared
+		@param group_id: id of the grop
+		@param member_id: id of the member the graph is shared
+	'''
+	# Create database connection
+	db_session = data_connection.new_session()
+
+	# Get the current time
+	cur_time = datetime.now()
+
+	new_event = models.ShareGraphEvent(graph_id=graph_id, owner_id=owner_id, group_id=group_id, member_id=member_id, share_time=cur_time, event_active=True)
+	db_session.add(new_event)
+	db_session.commit()
+	db_session.close()
+
+
+def update_share_graph_event(event_id, active, member_id):
+	'''
+		Update the share graph event. Change its active state.
+		If active is True then the notification is not read/clicked.
+
+		@param event_id: id of the share graph event
+		@param active: Boolean value, update the state of event
+		@param member_id: id of the user, the logged in user.
+	'''
+	db_session = data_connection.new_session()
+
+	try:
+		event = db_session.query(models.ShareGraphEvent).filter(models.ShareGraphEvent.id == event_id).filter(models.ShareGraphEvent.member_id == member_id).one()
+		event.event_active = active
+		db_session.commit()
+		db_session.close()
+	except NoResultFound:
+		db_session.close()
+		return 'Event not found'
```

Comment:
Raise an error instead of returning strings. 

There is way of documenting raised errors in python docstring. Please add that as well. The same goes with all methods. A method should return a single datatype or raise an error.